.github/workflows/west_cmds.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020 Linaro Limited.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Zephyr West Command Tests
+
+on:
+  push:
+    paths:
+    - 'scripts/west-commands.yml'
+    - 'scripts/west_commands/**'
+  pull_request:
+    paths:
+    - 'scripts/west-commands.yml'
+    - 'scripts/west_commands/**'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: cache-pip
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ matrix.python-version }}
+        restore-keys: |
+          ${{ runner.os }}-pip-${{ matrix.python-version }}
+    - name: install pytest
+      run: |
+        pip3 install pytest west
+
+    - name: run pytest
+      run: |
+        PYTHONPATH=./scripts/west_commands pytest ./scripts/west_commands/tests/
```

Comment:
Is this something you can customize on a per-platform basis, for instance?

Reply:
I'll look and see if I can.